code/datums/dna.dm

```diff
@@ -6,6 +6,7 @@
 	var/blood_type
 	var/datum/species/species = new /datum/species/human //The type of mutant race the player is if applicable (i.e. potato-man)
 	var/list/features = list("FFF") //first value is mutant color
+	var/uni_features ///Stores the hashed values of the person's non-human features.
```

Comment:
```suggestion
	var/list/features = list("FFF") //first value is mutant color
	///Stores the hashed values of the person's non-human features.
	var/uni_features
```
You need to put it over the variable, or it won't generate the doc properly.
The question remains unanswered, what's "uni"?